pkg/cloudprovider/providers/gce/gce_annotations.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gce
+
+import "k8s.io/kubernetes/pkg/api/v1"
+
+const (
+	// ServiceAnnotationLoadBalancerInternal is the annotation used on a service with type LoadBalancer
+	// to signify that an Internal LB should be assembled instead of an External LB.
+	// Value should be set to "true" (string) to enable.
+	ServiceAnnotationLoadBalancerInternal = "cloud.google.com/load-balancer-internal"
```

Comment:
BTW, we are trying to keep all API related annotations in https://github.com/kubernetes/kubernetes/blob/master/pkg/api/annotation_key_constants.go and https://github.com/kubernetes/kubernetes/blob/master/pkg/api/v1/annotation_key_constants.go. Is there any concern for moving it to the same place?